haskell_scotty/Server.hs

```diff
@@ -22,4 +22,9 @@ main = scotty 6969 $ do
     jsonAPI client
     status status201
 
+  delete "/api/clients/:id" $ do
+    clientId <- param "id"
+    liftIO $ deleteClient clientId
```

Comment:
@BartMassey I was a little surprised that I had to apply `liftIO` here, since `deleteClient` is a command, not a query, and we're not using the return value in any way. Can you shed any light here?